src/Symfony/Component/Translation/Bridge/Phrase/composer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "symfony/phrase-translation-provider",
+    "type": "symfony-translation-bridge",
+    "description": "Symfony Phrase Translation Provider Bridge",
+    "keywords": ["phrase", "translation", "provider"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "wicliff wolda",
+            "homepage": "https://github.com/wickedOne"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": "~8.1.0 || ~8.2.0",
+        "psr/cache": "^3.0",
```

Comment:
we're actually requiring a `psr/cache-implementation` here (same for the other abstracts deps)
but see my comment about cache+dispatcher before, because if we make this optional, this could stay as is